src/resources/animation-clip.js

```diff
@@ -0,0 +1,71 @@
+Object.assign(pc, function () {
+    'use strict';
+
+    /**
+     * @class
+     * @name pc.AnimationClipHandler
+     * @implements {pc.ResourceHandler}
+     * @classdesc Resource handler used for loading {@link pc.AnimationClip} resources.
+     */
+    var AnimationClipHandler = function () {
+        this.retryRequests = false;
+    };
+
+    Object.assign(AnimationClipHandler.prototype, {
+        load: function (url, callback) {
+            if (typeof url === 'string') {
+                url = {
+                    load: url,
+                    original: url
+                };
+            }
+
+            // we need to specify JSON for blob URLs
+            var options = {
+                retry: this.retryRequests
+            };
+
+            if (url.load.startsWith('blob:')) {
+                options.responseType = pc.Http.ResponseType.JSON;
+            }
+
+            pc.http.get(url.load, options, function (err, response) {
+                if (err) {
+                    callback(pc.string.format("Error loading animation clip resource: {0} [{1}]", url.original, err));
+                } else {
+                    callback(null, response);
+                }
+            });
+        },
+
+        open: function (url, data) {
```

Comment:
Might be good to document the anim clip json schema somewhere?

Reply:
Oh I see the schemas are already in PR description - awesome! I wonder where we could store them so we can reference and update them in future...